### PR TITLE
Fix Pylint 2.0 violations

### PR DESCRIPTION
**Pull request by netoarmando**
Fix the following violations aiming to support Pylint 2.0

- `unneeded-not` (C0113):
  Consider changing "not item in items" to "item not in items" used
  when a boolean expression contains an unneeded negation.

- `useless-import-alias` (C0414):
  Import alias does not rename original package Used when an import
  alias is same as original package.e.g using import numpy as numpy
  instead of import numpy as np

- `raising-format-tuple` (W0715):
  Exception arguments suggest string formatting might be intended Used
  when passing multiple arguments to an exception constructor, the
  first of them a string literal containing what appears to be
  placeholders intended for formatting

- `bad-continuation` (C0330):
  This was already included on the disable list, although with current
  version of pylint (2.0.0.dev2) violations at the end of the files
  are not being ignored.
  See: PyCQA/pylint#2278

- `try-except-raise` (E0705):
  The except handler raises immediately Used when an except handler
  uses raise as its first or only operator. This is useless because it
  raises back the exception immediately. Remove the raise operator or
  the entire try-except-raise block!

- `consider-using-set-comprehension` (R1718):
  Consider using a set comprehension Although there is nothing
  syntactically wrong with this code, it is hard to read and can be
  simplified to a set comprehension.Also it is faster since you don't
  need to create another transient list

- `dict-keys-not-iterating` (W1655):
  dict.keys referenced when not iterating Used when dict.keys is
  referenced in a non-iterating context (returns an iterator in
  Python 3)

- `comprehension-escape` (W1662):
  Using a variable that was bound inside a comprehension Emitted when
  using a variable, that was bound in a comprehension handler, outside
  of the comprehension itself. On Python 3 these variables will be
  deleted outside of the comprehension.

Issue: https://pagure.io/freeipa/issue/7614

Signed-off-by: Armando Neto <abiagion@redhat.com>

### CODE REVIEW

**Comment by netoarmando** [comment]
Travis is failing because its Pylint version doesn't recognize the new violations.

**Comment by tiran** [comment]
Unfortunately you have to catch ``bad-option-value`` while we support pylint 1.x.

File: ipalib/backend.py [hunk 1]
```diff
@@ -139,6 +139,7 @@ def execute(self, _name, *args, **options):
             if _name not in self.Command:
                 raise CommandError(name=_name)
             return self.Command[_name](*args, **options)
+        # pylint: disable=try-except-raise
```
[the remark this answers is not in the document]
Comment (netoarmando) [reply ipalib/backend.py]: I wasn't able to make this work, with `pylint 1.9.5`, it complains the same way, only adding `bad-option-value` to `pylintrc` changes the result. Submitting the changes anyway to see if travis acts the same (unexpected) way.

**Comment by tiran** [comment]
master:

* d13571942e41370fdbd2b6f9960c484fa61c3404 Fix Pylint 2.0 violations

**Pull request closed without merging**